consensus/istanbul/backend/engine_test.go

```diff
@@ -1229,6 +1229,51 @@ func TestSnapshot_Validators_AddRemove(t *testing.T) {
 	}
 }
 
+func TestSnapshot_Writable(t *testing.T) {
+	var configItems []interface{}
+	configItems = append(configItems, proposerPolicy(params.WeightedRandom))
+	configItems = append(configItems, epoch(3))
+	configItems = append(configItems, governanceMode("single"))
+	configItems = append(configItems, blockPeriod(0)) // set block period to 0 to prevent creating future block
+	chain, engine := newBlockChain(1, configItems...)
+
+	// add votes and insert voted blocks
+	var (
+		previousBlock, currentBlock *types.Block = nil, chain.Genesis()
+		err                         error
+	)
+
+	// voteData is inserted at block 4, and current block is block 5.
+	for i := 0; i < 5; i++ {
+		if i == 4 {
+			engine.governance.AddVote("governance.unitprice", uint64(2000000))
+		}
+		previousBlock = currentBlock
+		currentBlock = makeBlockWithSeal(chain, engine, previousBlock)
+		_, err = chain.InsertChain(types.Blocks{currentBlock})
+		assert.NoError(t, err)
+	}
+
+	// save current gov.changeSet's length for the expected value.
+	currentChangeSetLength := len(engine.governance.GetGovernanceChange())
```

Comment:
How about adding `assert.Equal(t, 1, currentChangeSetLength)` to explicitly show that currentChangeSetLength is not 0?

Reply:
thanks! reflected at https://github.com/klaytn/klaytn/pull/1470/commits/a06cb721e3901f5367926d9b943283da0cbd48da